vgscollect/src/main/java/com/verygoodsecurity/vgscollect/view/date/DatePickerBuilder.kt

```diff
@@ -30,7 +30,7 @@ internal class DatePickerBuilder(private val context: Context, mode:DatePickerMo
     private var listener: DatePicker.OnDateChangedListener? = null
     private var positiveListener: DialogInterface.OnClickListener? = null
     private var negativeListener: DialogInterface.OnClickListener? = null
-    private var visibilityChangeListener: ExpirationDateEditText.OnDatePickerVisibilityChangeListener? = null
+    private var visibilityChangeListener: OnDatePickerVisibilityChangeListener? = null
```

Comment:
@DonaldRG If `OnDatePickerVisibilityChangeListener` moved outside of `ExpirationDateEditText` it will cause breaching changes, because import will change.

Reply:
I moved it out because it is used in both components, DateRange and ExpirationDate. So it make sense to define it inside the parent of both. But you are right, I will leave it inside ExpirationDateEditText to avoid breaching changes. 
It will be weird to call it inside the implementation of DateRange, but I think it is ok.

Reply:
@DonaldRG We can make same interface for DateRange, and make them both inherited from internal interface, so we will not change imports and will not have code duplication.